backend/de.metas.adempiere.adempiere/base/src/main/java/de/metas/location/impl/AddressBuilder.java

```diff
@@ -530,32 +531,15 @@ private boolean isLocalCountry(final org.compiere.model.I_C_BPartner_Location lo
 		return isLocal;
 	}
 
-	private String getDisplaySequence(final boolean isLocal, final String trxName)
-	{
-		final Properties ctx = Env.getCtx();
-		final CountryCustomInfo userInfo = countriesRepo.retriveCountryCustomInfo(ctx, trxName);
-		String displaySequence = userInfo == null ? "" : userInfo.getCaptureSequence();
-		if (displaySequence == null || displaySequence.isEmpty())
-		{
-			final I_C_Country country = countriesRepo.getDefault(ctx);
-			displaySequence = getDisplaySequence(country, isLocal);
-		}
-		return displaySequence;
-	}
-
 	private void assertValidDisplaySequence(@NonNull final String displaySequence)
```

Comment:
not sure about the urgency, but i would suggest to do at least a follow up to call this `assertValidDisplaySequence` method when the country is saved. To tell the user ahead that the display sequence is wrong